src/main/scala/org/antipathy/scoozie/builder/TransitionBuilder.scala

```diff
@@ -1,16 +1,14 @@
 package org.antipathy.scoozie.builder
 
+import com.typesafe.config.Config
 import org.antipathy.scoozie.action._
 import org.antipathy.scoozie.action.control._
-import com.typesafe.config.Config
-import org.antipathy.scoozie.exception.UnknownActionException
-import scala.collection.immutable.Seq
-import org.antipathy.scoozie.configuration.Credentials
-import org.antipathy.scoozie.configuration.YarnConfig
-import scala.util.control.NonFatal
+import org.antipathy.scoozie.configuration.{Credentials, YarnConfig}
+import org.antipathy.scoozie.exception.{UnknownActionException, _}
+
 import scala.collection.JavaConverters._
-import com.typesafe.config.ConfigException
-import org.antipathy.scoozie.exception._
+import scala.collection.immutable.Seq
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Imports should be sorted alphabetically](https://app.codacy.com/app/SimonJPegg/scoozie/pullRequest?prid=3072461)